src/DOM/Classy/ParentNode.purs

```diff
@@ -38,12 +34,12 @@ childElementCount = PN.childElementCount <<< toParentNode
 
 -- | Finds the first child that is an element that matches the selector(s), or
 -- | null if no such element exists.
-querySelector :: forall n eff. IsParentNode n => String -> n -> Eff (dom :: DOM | eff) (Nullable N.Element)
-querySelector selector = PN.querySelector selector <<< toParentNode
+querySelector :: forall n eff. IsParentNode n => String -> n -> Eff (dom :: DOM | eff) (Maybe N.Element)
```

Comment:
Can we update the argument to use `QuerySelector` rather than `String` here (and below)?